lib/matplotlib/patches.py

```diff
@@ -181,14 +181,30 @@ def contains(self, mouseevent, radius=None):
 
     def contains_point(self, point, radius=None):
         """
-        Returns *True* if the given point is inside the path
+        Returns `True` if the given *point* is inside the path
         (transformed with its transform attribute).
+
+        *radius* allows the path to be made slightly larger or smaller.
         """
         radius = self._process_radius(radius)
         return self.get_path().contains_point(point,
                                               self.get_transform(),
                                               radius)
 
+    def contains_points(self, points, radius=None):
+        """
+        Returns a bool array which is `True` if the (closed) path
+        contains the corresponding point.
+        (transformed with its transform attribute).
+
+        *points* should be an N x 2 array.
```

Comment:
Do you need to check for the right shape before sending to contains_points?  Can you pass a 2xN array?  Thats what Id usually pass...

Reply:
@jklymak This PR only exposes the `Path.contains_points()` method, so I think further improvement to this method should be separately made to https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/path.py#L501

Reply:
Yes but the docs shouldn't say something vague like "usually"

Reply:
@jklymak that's fair. I changed to "must be", to be consistent with what `Path.contains_points()` raises when 2xN array is passed.